.github/workflows/gobuild.yml

```diff
@@ -35,15 +35,23 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  generate-model:
     name: Kubernetes Model Generator Build
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Why are we changing CI builder to MacOS instead of ubuntu?

I'm not able to understand why is this change required.

Reply:
Performance. Pipeline runs 10 minutes faster in mac arm64